### PR TITLE
Feat: added pagination

### PR DESCRIPTION
**Pull request by ZixMai**
1. added pagination to requests
2. added more fields to protobuf file for pagination

### CODE REVIEW

File: event_service/proto/event_service.proto [hunk 1]
```diff
@@ -68,6 +72,8 @@ message GrpcUser {
 
 message RequestingUser {
   GrpcUser user = 1;
+  int32 page_number = 3;
```
Comment (Nhsdkk) [inline event_service/proto/event_service.proto]: Use int64

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Check my review

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Review by MaxPlays35 — APPROVED** [review]

File: event_service/proto/event_service.proto (hunk 1 above)
Comment (Nhsdkk) [reply event_service/proto/event_service.proto]: Fixed

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**